infrastructure/template.yaml

```diff
@@ -1190,14 +1220,14 @@ Resources:
       LogGroupName: !Ref RedactedBearerTokenRetrievalStateMachineLogGroup
       DestinationArn: !FindInMap [PlatformConfiguration, !Ref Environment, CSLSEGRESS]
 
-  BearerTokenRetrievalStateMachineFailedMetric:
+  BearerTokenRetrievalStateMachineTaskFailedMetric:
     Type: AWS::Logs::MetricFilter
     Properties:
       LogGroupName: !Ref BearerTokenRetrievalStateMachineLogGroup
-      FilterPattern: '{($.type = "ExecutionFailed")}'
+      FilterPattern: '{$.type = "TaskFailed"}'
```

Comment:
How come we changed this? My understanding is that `TaskFailed` doesn't cover timeouts?

Reply:
Ignore me you've got `BearerTokenRetrievalStateMachineExecutionFailedCanaryAlarm